reports/California.md

```diff
@@ -141,6 +140,22 @@ id: ca-longbeach-1
 
 ## Los Angeles
 
+### Protestor shot in head | (believed to be) May 30th
+
+Footage shows a protestor sitting on the curb. They are bleeding from a wound to the head. Other protestors are attempting to treat them. Greg Doucette says that, per the journalist who submitted the video, police shot first, then gave the dispersal order.
+
+tags: shoot, projectiles
+
+id: ca-losangeles-30
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1268052132252602368
+* https://twitter.com/chick_in_kiev/status/1266867554041188353
```

Comment:
Am I misunderstanding? This just seems like a megathread on antifa stuff?

Reply:
Quite correct. Thanks for noticing. I had two tweets open in different tabs and must have copy-pasted a link to the wrong one. Correcting.